exercise_2/openweather_api_client.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import collections
+
+
+WeatherForecast = collections.namedtuple(
+    "WeatherForecast", ("city", "country", "current_weather", "temperature", "pressure")
+    )
+
+
+class BasicWeatherApiClient:
+    """Basic abstraction for services to handle serializing data from service
+      and wrap them around data class.
+    """
+
+    city = None
```

Comment:
To nie jest do końca dobrze użyte. A co jeśli poprosiłbym, że chcielibyśmy by API wspierało szukanie po lokalizacji ?